docs/oathkeeper/guides/websockets.mdx

```diff
@@ -0,0 +1,360 @@
+services:
+  oathkeeper:
+    image: oryd/oathkeeper:${useLatestRelease('oathkeeper')}
+    depends_on:
+      - kratos
+    ports:
+      - 8080:4455
+      - 4456:4456
+    command:
+      serve proxy -c "/etc/config/oathkeeper/oathkeeper.yml"
+    environment:
+      - LOG_LEVEL=debug
+    restart: on-failure
+    networks:
+      - intranet
+    volumes:
+      - ./oathkeeper:/etc/config/oathkeeper
+  postgres-kratos:
+    image: postgres:9.6
+    environment:
+      - POSTGRES_USER=kratos
+      - POSTGRES_PASSWORD=secret
+      - POSTGRES_DB=kratos
+    networks:
+      - intranet
+  kratos-migrate:
+    image: oryd/kratos:${useLatestRelease('kratos')}
+    links:
+      - postgres-kratos:postgres-kratos
+    environment:
+      - DSN=postgres://kratos:secret@postgres-kratos:5432/kratos?sslmode=disable&max_conns=20&max_idle_conns=4
+    networks:
+      - intranet
+    volumes:
+      - type: bind
+        source: ./kratos
+        target: /etc/config/kratos
+    command: -c /etc/config/kratos/kratos.yml migrate sql -e --yes
+  kratos:
+    image: oryd/kratos:${useLatestRelease('kratos')}
+    links:
+      - postgres-kratos:postgres-kratos
+    environment:
+      - DSN=postgres://kratos:secret@postgres-kratos:5432/kratos?sslmode=disable&max_conns=20&max_idle_conns=4
+    ports:
+      - '4433:4433'
+      - '4434:4434'
+    volumes:
+      - type: bind
+        source: ./kratos
+        target: /etc/config/kratos
+    networks:
+      - intranet
+    command: serve -c /etc/config/kratos/kratos.yml --dev --watch-courier
+  kratos-selfservice-ui-node:
+    image: oryd/kratos-selfservice-ui-node:latest
+    environment:
+      - KRATOS_PUBLIC_URL=http://kratos:4433/
+      - KRATOS_BROWSER_URL=http://127.0.0.1:4433/
+    networks:
+      - intranet
+    ports:
+      - "4455:3000"
+    restart: on-failure
+  mailslurper:
+    image: oryd/mailslurper:latest-smtps
+    ports:
+      - '4436:4436'
+      - '4437:4437'
+    networks:
+      - intranet
+networks:
+  intranet:`}</CodeBlock>
+
+This example uses the following network architecture:
+
+- `4433` port is a public ("Browser") API of Ory Kratos
+- `4434` is an admin API of Ory Kratos
+- `4455` is a port of the User Interface implemented by self-service-ui
+- `8080` is a port of Ory Oathkeeper
+
+Other ports and services are available only in the internal network.
+
+## Configuring Ory Oathkeeper and Ory Kratos
+
+1. Let's create a `kratos` folder and fetch configuration files
+
+<CodeBlock className="language-shell">{`mkdir kratos
+wget https://raw.githubusercontent.com/ory/kratos/${useLatestRelease('kratos')}/contrib/quickstart/kratos/email-password/identity.schema.json -O kratos/identity.schema.json
+wget https://raw.githubusercontent.com/ory/kratos/${useLatestRelease('kratos')}/contrib/quickstart/kratos/email-password/kratos.yml -O kratos/kratos.yml`}</CodeBlock>
+
+2. Create `oathkeeper` folder and `oathkeeper/oathkeeper.yml` with the following content:
+
+```yaml
```

Comment:
Should we move these code snippets to files in the code-examples directory? https://github.com/ory/docs/tree/master/code-examples

That would allow us to also run some tests against it!

Reply:
All the other examples we have created are tested continously and ensure that the things we write in the docs actually work. Broken examples are the #1 frustration for developers!

I am fine to merge this, but let's not make this a "merge and forget" thing. If you link a follow up with description & assignee I am fine to merge this as it is now

Reply:
Fair point. I will talk to Andrew and see if he can add tests soon or if we should track it for later.

Reply:
I moved everything to `code-examples` and it works fine without any issues. I added test case for unauthenticated requests using Go. Do I need to use a playwright to write tests against the example?